src/Entity/Trick.php

```diff
@@ -200,4 +187,16 @@ public function removeImage(Image $image): self
         return $this;
     }
 
+    public function getMainImage(): ?Image
+    {
+        return $this->mainImage;
+    }
+
+    public function setMainImage(?Image $mainImage): self
+    {
+        $this->mainImage = $mainImage;
+
+        return $this;
+    }
+
```

Comment:
public function setMainImageWithFirstImage(): void
    {
        $this->mainImage = $this->images->first();
    }